ale_linters/eruby/erb.vim

```diff
@@ -0,0 +1,24 @@
+" Author: Matthias Guenther - https://wikimatze.de, Eddie Lebow https://github.com/elebow
+" Description: ERB from the Ruby standard library, for eruby/erb files
+
+function! ale_linters#eruby#erb#GetCommand(buffer) abort
+    let l:rails_root = ale#ruby#FindRailsRoot(a:buffer)
+
+    if l:rails_root is? ''
+        return 'erb -P -x %t | ruby -c'
+    endif
+
+    " Rails-flavored eRuby does not comply with the standard as understood by
+    " ERB, so we'll have to do some substitution. This does not reduce the
+    " effectiveness of the linter—the translated code is still evaluated.
+    return 'ruby -rerb -e "puts ERB.new(File.read(%t, encoding: \"BINARY\").gsub(\"<%=\",\"<%\"), nil, \"-\").src" | ruby -c'
+endfunction
+
+call ale#linter#Define('eruby', {
+\   'name': 'erb',
+\   'executable': 'erb',
+\   'output_stream': 'stderr',
+\   'command_callback': 'ale_linters#eruby#erb#GetCommand',
+\   'callback': 'ale#handlers#ruby#HandleSyntaxErrors',
+\})
```

Comment:
There's an undocumented feature for backwards compatibility you should use here. Add `'aliases': ['erubylint'],` to the dictionary here. Then the linter will continue to work for uses who have configured `g:ale_linters` with the old name. You can see how it works in `autoload/ale/linter.vim`.